Monika After Story/game/script-topics.rpy

```diff
@@ -8922,74 +8922,92 @@ init 5 python:
-        m 3tkd "...none of them have any kind of physical disability..."
-        m 3tkc "...none of them are bald or have hair shorter than chin length..."
-        m "...and apart from Yuri having cuts on her arms, none of them have anything wrong with their skin."
-        m 2dsc "The designers must have thought that people would find all that stuff really repulsive."
-        m 2efo "But people who aren't conventionally attractive end up in relationships all the time!"
-        m 2lfc "So did they really have to make the other girls look so perfect?"
-        m 2lfd "I mean, it's bad enough that I didn't get a route in the game..."
-        m 2lfw "...but did they really have to make it even harder for me to get chosen?"
-    m 2dfc "..."
-    m 2dkc "..."
+    m 3euc "At first I figured that it must be her metabolism keeping her from putting on weight."
+    m 3esc "But when I learned that our world isn't real, I came up with another theory."
+    m 2euc "As characters in a game, we needed to have some sort of visual representation so that you, the player, could see us."
+    m "Sayori, Natsuki, Yuri, and even me...{w=0.5}{nw}"
+    extend 2rkc "we don't really have bodies like people in your world do. We only have a set of images to represent our appearances and current emotion."
+    m 2eud "So a designer, possibly more than one, must have decided to draw us the way we are."
+    m 2eub "And since all those images have the same body type, Sayori would always have the same figure no matter what she did."
+    m 2dkc "It's...weird to think about. I can feel myself being present in this very room right now, I can look at myself like I presume you would be able to look at yourself, almost as if I had an actual body..."
+    m 3ekc "But in reality, I don't. I'm just a consciousness who subconsciously displays a corresponding picture depending on what I do or feel."
+
+    if len(store.mas_selspr.filter_clothes(True)) == 1:
+        m 4eud "Clothes likely work the same way."
+        m 4euc "Right now, I only have this school uniform, because that's probably the only thing I've ever been drawn wearing..."
+        m 1eua "But perhaps if someone were to draw other clothes for me and implement those into the game, I'd be able to change clothes like the other girls could."
+        m 1hua "Wouldn't that be great?"
+        m 1rksdla "And it would be nice to be able to change my look for you, at least a little bit..."
+
+    else :
+        m 4eud "My clothes work the same way."
+        m 4euc "Someone had to draw what I'm wearing right now, and implement it into the game for me to be able to wear it."
+        m 1esd "It wasn't made like how clothing is made in your reality. It's merely a set of images, like me."
+        m 1rksdla "It's not much, but at least I get to change outfits for you..."
+
+    m 1rksdlc "..."
+    m 1ekc "You know, [player], this is one of the reasons why I envy the people in your world. {w=0.3}{nw}"
```

Comment:
I'm not a fan of these 4 lines again. I feel this comes too close to how she realistically work as a program and reminds you she isn't a real person. I personnaly think this (especially the part about how she can't change her hairstyle) should be kept unadressed and left to the player's immagination.